.env.virtual-funding

```
@@ -31,3 +31,8 @@ REACT_APP_TRACKER_URL = 'localhost:8000'
 REACT_APP_TRACKER_URL_HTTP_PROTOCOL = 'http'
 REACT_APP_WALLET_URL = 'http://localhost:3055'
 SKIP_PREFLIGHT_CHECK = 'true'
+
+
+## e2e-tests
+HEADLESS=true
```

I actually prefer to run e2e tests with `HEADLESS = false` locally. Also, a nit on formatting
```suggestion
HEADLESS = `true`
```